CMakeLists.txt

```
@@ -122,19 +122,20 @@ if 'real_prefix' in dir(sys):
   endif ()
 
   # boost-python
-  set (BOOST_PYTHON_COMPONENT_NAMES
-       python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
-       python${PYTHON_VERSION_MAJOR} python)
-  foreach (NAME ${BOOST_PYTHON_COMPONENT_NAMES})
-    if (NOT Boost_FOUND)
-      message (STATUS "Looking for component ${NAME}")
-      find_package (Boost QUIET COMPONENTS ${NAME})
-    endif ()
-  endforeach ()
+  #set (BOOST_PYTHON_COMPONENT_NAMES
+  #     libboost-python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
+  #     python${PYTHON_VERSION_MAJOR} python)
+  #foreach (NAME ${BOOST_PYTHON_COMPONENT_NAMES})
+  #  if (NOT Boost_FOUND)
+  #    message (STATUS "Looking for component ${NAME}")
+  #    find_package (Boost QUIET COMPONENTS ${NAME})
+  #  endif ()
+  #endforeach ()
+  FIND_PACKAGE(Boost COMPONENTS python)
   if (Boost_FOUND)
     set (HAVE_BOOST_PYTHON TRUE)
     if (NOT TARGET Boost::python)
-      add_library (Boost::python UNKNOWN IMPORTED)
+      add_library (Boost::python SHARED IMPORTED)
       target_include_directories (Boost::python INTERFACE ${Boost_INCLUDE_DIRS})
       set_target_properties (Python::Module PROPERTIES IMPORTED_LOCATION
                                                        ${Boost_LIBRARIES})
```

Here we need the same thing again:
```suggestion
      set_target_properties (
        Boost::python
        PROPERTIES IMPORTED_LOCATION ${Boost_LIBRARIES}
                   INTERFACE_INCLUDE_DIRECTORIES ${Boost_INCLUDE_DIRS})
```
And you will also need to do an analogous fix in FindNumpy.cmake to replace
```cmake
  target_include_directories (Python::NumPy INTERFACE ${NumPy_INCLUDE_DIRS})
```
by 
```cmake
  set_target_properties (Python::NumPy PROPERTIES INTERFACE_INCLUDE_DIRECTORIES
                                                  ${NumPy_INCLUDE_DIRS})
```